cli/jobs/train/r/README.md

```diff
@@ -0,0 +1,40 @@
+---
+page_type: sample
+languages:
+- R
+- azurecli
+products:
+- azure-machine-learning
+description: Learn how to submit R jobs to an AzureMLCompute Cluster using the AzureML 2.0 CLI.
+---
+
+# R using the AzureML 2.0 CLI
```

Comment:
AzureML -> Azure ML

maybe reword to "Train machine learning models with R using the 2.0 CLI for Azure ML"?